config/routes.rb

```diff
@@ -1,9 +1,11 @@
 Rails.application.routes.draw do
   devise_for :users
-  resources :restaurants, only:
-  [
+  root 'restaurants#index'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.